exo-deploy/terraform/aws/alb/route53.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_route53_record" "external" {
```

Comment:
That would allow for finer grained access control. At the very least, we should assign the private security group to private services & workers and the public one to public services, otherwise there isn't much of a point in having two of them